modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/search/spi/model/index/contributor/DDMStructureModelDocumentContributor.java

```diff
@@ -61,15 +62,18 @@ public void contribute(Document document, DDMStructure ddmStructure) {
 
 			document.addKeyword(Field.STATUS, structureVersion.getStatus());
 			document.addKeyword(Field.VERSION, structureVersion.getVersion());
+
+			document.addKeyword(
```

Comment:
@rodrigopaulino is the order important? I mean, can we move to before line 63?

Reply:
I've moved to other try/catch block, as an exception might be thrown in the previous lines.